libnopegl/src/node_renderother.c

```diff
@@ -210,6 +215,31 @@ struct rendertexture_opts {
     struct render_common_opts common;
 };
 
+struct rendernoise_priv {
+    struct render_common common;
+};
+
+struct rendernoise_opts {
+    int type;
+    struct ngl_node *frequency_node;
+    float frequency;
+    struct ngl_node *amplitude_node;
+    float amplitude;
+    struct ngl_node *octaves_node;
+    int32_t octaves;
```

Comment:
`uint32_t`, or change the parameter option (and maybe how it's pushed to the pipeline)

Reply:
Done.